### PR TITLE
Replace plugin Jenkinsfile by a library one.

### PR DESCRIPTION
**Pull request by oleg-nenashev**
buildPlugin() does not longer work for libraries, so let’s use a manually-written Jenkinsfile for now.

P.S: I will get rid of this copy-paste next week

@reviewbybees @jglick

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1 +1,56 @@
...
+                            else {
+                                bat command
+                            }
+                        }
+                    }
+                }
+
+                stage('Archive') {
+                    /* Archive the test results */
+                    junit '**/target/surefire-reports/TEST-*.xml'
```
Comment (dwnusbaum) [inline Jenkinsfile]: I don't see the test results in the archive of the CI build, is the glob pattern correct? I am looking at https://ci.jenkins.io/job/jenkinsci-libraries/job/lib-jenkins-maven-embedder/job/PR-16/.

**Comment by oleg-nenashev** [comment]
@dwnusbaum I'd guess you're just looking in the BlueOcean interface :) https://ci.jenkins.io/job/jenkinsci-libraries/job/lib-jenkins-maven-embedder/job/PR-16/1/testReport/hudson.maven/

**Comment by dwnusbaum** [comment]
@oleg-nenashev No I was looking at the classic UI but in the wrong place :man_facepalming:

**Comment by oleg-nenashev** [comment]
@reviewbybees done. Will merge it, it definitely does not make the situation worse

**Pull request merged**